clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -875,6 +876,11 @@ void EmitAssemblyHelper::RunOptimizationPipeline(
 
   ModulePassManager MPM;
 
+  // FIXME: Change this when -fno-sycl-early-optimizations is not tied to
```

Comment:
Is `fno-sycl-early-optimizations` going to be **not** tied to `disable-llvm-passes`?

Reply:
Optimally, I believe so. @AlexeySachkov - Do you know what the plans are here exactly?

Reply:
Because if there is no such plan, we don't need the FIXME.

Reply:
We have an internal tracker for that and it falls into a category of our technical debt. So I would say it should be ok to have this FIXME here